app/views/filters/index.html.erb

```diff
@@ -1,3 +1,48 @@
 <%= turbo_frame_tag 'frame_filters' do %>
-  <p>You have a total of <%= keywords_count %> keywords.</p>
+  <p>
+    <%= t('filters.total_of_keywords', keywords_count: keywords_count) %>
+  </p>
+  <form action="<%= keywords_path %>" method="GET" class="form-filters" data-turbo="true" data-turbo-frame="frame_list_keywords">
+    <div>
+      <label for="keyword_pattern" class="form-label">
+        <i class="bi bi-list-check"></i> <%= t('filters.by_keyword') %>...
+      </label>
+      <input type="text" name="keyword_pattern" placeholder="type a keyword pattern" class="form-control"/>
```

Comment:
How will we populate the value if user visits directly by url? ```localhost:3000/keywords?keyword_pattern=pattern`

Reply:
@ankitkalia1195, it is possible to update the code and pass the params into the filter Turbo Frame `src` attribute, similar to this:
![image](https://user-images.githubusercontent.com/77609814/127103244-a0f5ea79-6c15-424b-91e2-1eb30ca88404.png)
(this partial view is built from keywords#index action, so it is easy to provide it with the local `filter_params`)

I did not implement it at the moment as it is not part of the specifications to save a filtered URL but I will probably make it as a nice-to-have to try & confirm if it works or not ;-)

Reply:
Issue added here:
- #87 
- #88